humanlayer-wui/src/components/internal/SessionDetail/components/ForkViewModal.tsx

```diff
@@ -235,7 +256,7 @@ function ForkViewModalContent({
 
         <div className="flex items-center justify-between text-xs text-muted-foreground mt-4 pt-4 border-t">
           <div className="flex items-center gap-4">
-            <span>↑↓ j/k Navigate</span>
+            <span>↑↓/j/k Navigate</span>
```

Comment:
whats the purpose of this change? 


```suggestion
            <span>↑↓ j/k Navigate</span>
```